core/src/main/java/io/kestra/core/serializers/ListOrMapOfLabelDeserializer.java

```diff
@@ -26,7 +26,14 @@ public List<Label> deserialize(JsonParser p, DeserializationContext ctxt) throws
         else if (p.hasToken(JsonToken.START_ARRAY)) {
             // deserialize as list
             List<Map<String, String>> ret = ctxt.readValue(p, List.class);
-            return ret.stream().map(map -> new Label(map.get("key"), map.get("value"))).toList();
+            return ret.stream().map(map -> {
```

Comment:
Could it be replaced by `.map(this::validateAndCreateLabel)`?

Also adding a test would be great